core/src/storage/impls/storage_db/kvdb_sqlite.rs

```diff
@@ -169,7 +174,7 @@ impl<ValueType> KvdbSqlite<ValueType> {
             connection
                 .execute(
                     &statements.create_table_number_key,
```

Comment:
Here we should create a new set of statements for the number key table. Save it in the main statements variable.